pdp/static/js/pdp_controls.js

```diff
@@ -179,6 +180,35 @@ Colorbar.prototype = {
         }
     },
 
+    round_to_nearest: function (mult) {
+      this.minimum = Math.round(this.minimum / mult) * mult;
```

Comment:
Again, I think that this function should be structured as a take_parameter/return_value, rather than a side-effects based function that modifies state. Since you're performing the exact operation for each value, why not just pass the value in as a parameter and return the result? E.g.

```javascript
round_to_nearest: function (value, mult) {
    return Math.round(value / mult) * mult;
},
```